{{Introduction to Python Basics: Variables and Data types}}/.md

```diff
@@ -0,0 +1,180 @@
+
+c.Strings:
+
+The final Data Types that we will be talking about are strings. Strings are described as a sequence of characters enclosed by quotation marks. These characters can be word or even numbers. The quotation marks can either be single quotes( ‘’) or double quotes (“”) it doesn't matter which you decide to choose however you must be uniform with whichever you choose. 
+
+Ex 1.4: 
+
+	string = 'Welcome to this blog!'
+
+This is the correct way to use single quotes. You can notice that the string starts with a single quote and ends with a single quote which is what we want to do.
+
+Ex 1.5: 
+
+	string = 'Welcome to this blog!"
+
+Python will not like this string because we do not end with the same type of quote that we started with, we aren't being uniform. The string begins with a single quote and ends with a double quote. If we start with a single quote we must end with a single quote. 
+
+Ex 1.6:
+
+	string = '42'
+
+This is another example of a string. Although we would think that ‘42’ is an integer that is not the case for this particular example. Since the number 42 is in quotation marks it is actually considered a string. Any numbers that are enclosed in quotation marks are considered string. 
+
+Ex 1.7:
+
+	string = '42.0'
+
+This example is similar to the example above except the value 42.0 is considered a floating number because of the decimal. However, the same logic is applied to this example. Since the value is enclosed in single quotation marks the floating number 42.0 is actually the string ‘42.0’.
+
+Test your knowledge:
+
+	32.189 is considered what kind of Data Type?
+	
+	‘90’ is a number. True or False?
+	
+	“Hello, what is your name?’ is an acceptable string. True or False?
+	
+	-78 is what kind of Data Type?
+
+
+
+
+II. Variables
+
+Now that we have learned about Data Types we will dive into variables. I want you to think of Python variables like a box. The box itself is the variable and you can place a single value into the box. The values stored inside can be any of the Data Types that we previously talked about (Integers, Floating numbers or Strings)
+
+[Image represents box analogy](https://www.hackerearth.com/practice/python/getting-started/python-variables/tutorial/)
+
+(In this picture, the box represents the variable named pi.Inside the variable pi we have stored the value 3.14.)
+
+
+a.Naming a variable:
+
+Naming a variable is important because it gives context to the store valuable that it represents. A variable can contain special characters, numbers and letters. The only thing to remember when naming a variable is that it cannot begin with a number or contain python keywords(refer to chart below). Once you name a variable you assign it to a single value using an equal sign. To the left of the equal sign is the variable name and to the right is the value stored in that variable. 
```

Comment:
clarification for naming restriction
```suggestion
Naming a variable is important because it gives context to the stored value that it represents. A variable can contain special characters, numbers, and letters. When naming a variable, know that it cannot begin with a number or contain python keywords (refer to the chart below). Once you name a variable you assign it to a single value using an equal sign. To the left of the equal sign is the variable name and to the right is the value stored in that variable.
```